components/org.wso2.carbon.identity.oauth.dcr/src/main/java/org/wso2/carbon/identity/oauth/dcr/service/DCRMService.java

```diff
@@ -74,7 +74,8 @@
      */
     public Application getApplication(String clientId) throws DCRMException {
 
-        return buildResponse(getApplicationById(clientId));
+        boolean isApplicationRolePermissionRequired = DCRMUtils.isApplicationRolePermissionRequired();
+        return buildResponse(getApplicationById(clientId, isApplicationRolePermissionRequired));
```

Comment:
`return buildResponse(getApplicationById(clientId,DCRMUtils.isApplicationRolePermissionRequired()));`

Reply:
Fixed in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1461